src/branch_fixer/services/git/pr_manager.py

```
@@ -57,13 +59,44 @@ def create_pr(self,
             PRValidationError: If branch doesn't exist or has conflicts
         """
         pr_id = len(self.prs) + 1
+        url: Optional[str] = None
+
+        if shutil.which("gh"):
+            try:
+                result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)